python_binding/lsm/_client.py

```diff
@@ -1427,3 +1427,320 @@ def volume_ident_led_clear(self, volume, flags=FLAG_RSVD):
+                a write process.
+                For SSD's physical disk cache, please check with the vendor of
+                your hardware RAID card and SSD disk.
+                Valid values are:
+                    * lsm.Volume.PHYSICAL_DISK_CACHE_ENABLED
+                        Physical disk cache enabled.
+                    * lsm.Volume.PHYSICAL_DISK_CACHE_DISABLED
+                        Physical disk cache disabled.
+                    * lsm.Volume.PHYSICAL_DISK_CACHE_USE_DISK_SETTING
+                        Physical disk cache is determined by the disk vendor
+                        vis physical disks' SCSI caching mode page(0x08 page).
+                        It's strongly suggest to change this value to
+                        lsm.Volume.PHYSICAL_DISK_CACHE_ENABLED or
+                        lsm.Volume.PHYSICAL_DISK_CACHE_DISABLED
+                    * lsm.Volume.PHYSICAL_DISK_CACHE_UNKNOWN
+                        Plugin failed to detect the physical disk status.
+        SpecialExceptions:
+            LsmError
+                ErrorNumber.NO_SUPPORT
+                ErrorNumber.NOT_FOUND_VOLUME
+        Capability:
+            lsm.Capabilities.VOLUME_CACHE_INFO
+        """
+        return self._tp.rpc('volume_cache_info', _del_self(locals()))
+
+    @_return_requires(None)
+    def volume_physical_disk_cache_set(self, volume, pdc, flags=FLAG_RSVD):
+        """
+        lsm.Client.volume_physical_disk_cache_set(self, volume, pdc,
+                                                  flags=lsm.Client.FLAG_RSVD)
+
+        Version:
+            1.3
+        Usage:
+            Change the setting of RAM physical disk cache of certain volume.
+            On some product(like HP SmartArray), this action will be effective
+            at system level which means even you are requesting a change
+            on certain volume, it still impact on all other volumes on the same
+            controller(system).
+        Parameters:
+            volume (Lsm.Volume)
+                The lsm.Volume instance.
+            pdc (int)
+                lsm.Volume.PHYSICAL_DISK_CACHE_ENABLED
+                    Enable physical disk cache.
+                lsm.Volume.PHYSICAL_DISK_CACHE_DISABLED
+                    Disable physical disk cache
+            flags (int, optional):
+                Reserved for future use. Should be set as lsm.Client.FLAG_RSVD
+        Returns:
+            N/A
+        SpecialExceptions:
+            LsmError
+                ErrorNumber.NO_SUPPORT
+                    You might also get NO_SUPPORT error when trying
+                    to change SSD physical disk cache on MegaRAID.
+                ErrorNumber.NOT_FOUND_VOLUME
+        Capability:
+            lsm.Capabilities.VOLUME_PHYSICAL_DISK_CACHE_SET
+                Allow changing physical disk cache.
+            lsm.Capabilities.VOLUME_PHYSICAL_DISK_CACHE_SET_SYSTEM_LEVEL
+                Indicate this action will change system settings which
```

Comment:
+                Indicates that this action will change system settings which
 +                are effective for all volumes in this system.
 +                For example, on HPE SmartArray, the physical disk cache
 +                setting is a controller level setting.